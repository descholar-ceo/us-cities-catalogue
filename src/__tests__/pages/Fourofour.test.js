import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Fourofour from '../../pages/Fourofourpage';

describe('Test Fourofour component', () => {
  it('Renders the Fourofour component', done => {
    const FourofourComp = shallow(<Fourofour />);
    expect(FourofourComp.type()).toBe('div');
    expect(FourofourComp.find('p').exists()).toBe(true);
    done();
  });
});

describe('Test the snapshot of Fourofour component', () => {
  it('Render the correct snapshot', done => {
    const FourofourSnapshot = renderer.create(<Fourofour />).toJSON();
    expect(FourofourSnapshot).toMatchSnapshot();
    done();
  });
});
