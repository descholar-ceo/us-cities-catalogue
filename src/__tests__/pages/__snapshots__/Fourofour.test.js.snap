// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the snapshot of Fourofour component Render the correct snapshot 1`] = `
<div>
  <p>
    Ooops!!! The page you requested does not exist, sorry!
  </p>
</div>
`;
